pkg/worker/handler/keypair/handler.go

```diff
@@ -32,13 +39,19 @@ func New(c Config) *Handler {
 	}
 
 	cou := map[string]recorder.Interface{}
-	gau := map[string]recorder.Interface{}
-	his := map[string]recorder.Interface{}
 
 	{
-		// metrics can be registered here
+		nam := "aws_ec2_keypair_total"
```

Comment:
what happens if this string were misspelled here (or in the other file it's used)?

just wondering if it would be helpful to have this in a constant, or there's some kind of typecheck or something that would catch that.